client/post-editor/editor-gutenberg-blocks-warning-dialog/index.jsx

```diff
@@ -128,6 +138,45 @@ class EditorGutenbergBlocksWarningDialog extends Component {
 	}
 }
 
+const mapDispatchToProps = dispatch => ( {
+	optIn: ( siteId, gutenbergUrl ) => {
```

Comment:
This should be `switchToGutenberg`, otherwise [this line](https://github.com/Automattic/wp-calypso/pull/28525/files#diff-3e798fad7f49f912349ef6ea5ae179e9R85) will fail.
```suggestion
	switchToGutenberg: ( siteId, gutenbergUrl ) => {
```